internal/cmd/controller/gitops/reconciler/status_controller.go

```diff
@@ -179,6 +179,11 @@ func (r StatusReconciler) setReadyStatusFromBundle(ctx context.Context, gitrepo
 		return err
 	}
 
+	// Make sure the bundles are always iterated in the same order
```

Comment:
```suggestion
	// Make sure the bundles are always iterated in the same order
	// The code below will pick the first element matching the condition, so successive executions should produce the same result.
```

Reply:
Thanks, lgtm. I can't review the PR (I don't know why).